routing/localchans/manager.go

```diff
@@ -14,6 +15,9 @@ import (
 // Manager manages the node's local channels. The only operation that is
 // currently implemented is updating forwarding policies.
 type Manager struct {
+	// Db is used to query local channel parameters
+	Db *channeldb.DB
```

Comment:
Similar comment here re slimmer interface or a function closure as it'll make this logic easier to test in the future.